src/ringcentral-web-phone.js

```diff
@@ -1034,6 +995,73 @@
         return sendReceive(this, messages.park);
     }
 
+    /*--------------------------------------------------------------------------------------------------------------------*/
+    /**
+     * @this {SIP.Session}
+     * @return {Promise}
+     */
+
+    function reinvite (options, modifier){
+        var session = this;
+        options = options || {}
+        options.sessionDescriptionHandlerOptions = options.sessionDescriptionHandlerOptions || {};
+        options.sessionDescriptionHandlerOptions.constraints = options.sessionDescriptionHandlerOptions.constraints || {audio: true, video: false};
+        return session.__reinvite(options, modifier);
+    }
+
+    /*--------------------------------------------------------------------------------------------------------------------*/
+
+
+    function toggleMute (session , mute) {
+        var pc = session.sessionDescriptionHandler.peerConnection;
+        if (pc.getSenders) {
+            pc.getSenders().forEach(function(sender) {
+                if (sender.track) {
+                    sender.track.enabled = !mute;
+                }
+            });
+        }
+    };
+
+    /*--------------------------------------------------------------------------------------------------------------------*/
+    function mute (silent){
+        if (this.state !== this.STATUS_CONNECTED) {
+            this.logger.warn('An acitve call is required to mute audio');
+            return;
+        }
+        this.logger.log('Muting Audio');
+        if (!silent) {
+            this.emit('muted',this.session);
+        }
+        return toggleMute(this, true);
+    };
+
+    /*--------------------------------------------------------------------------------------------------------------------*/
+
+    function unmute(silent) {
+        if (this.state !== this.STATUS_CONNECTED) {
+            this.logger.warn('An active call is required to unmute audio');
+            return;
+        }
+        this.logger.log('Unmuting Audio');
+        if (!silent) {
+            this.emit('unmuted',this.session);
+        }
+        return toggleMute(this,false);
+    };
+
+    /*--------------------------------------------------------------------------------------------------------------------*/
+
+    /**
+     * @this {SIP.Session}
+     * @return boolean
+     */
+
+    function onHold (){
```

Comment:
Rename to setLocalHold

Reply:
Renamed setHold function to setLocalHold